src/mumble/Settings.h

```diff
@@ -431,10 +431,15 @@ struct Settings {
 	QByteArray qbaPTTButtonWindowGeometry = {};
 
 	// Network settings
-	bool bTCPCompat   = false;
-	bool bReconnect   = true;
-	bool bAutoConnect = false;
-	bool bQoS         = true;
+	bool bTCPCompat                   = false;
```

Comment:
I know that this setting existed before, but do you happen to know what it is for? Do we still need that?

Reply:
It is the flag that represents "Forced TCP mode" from the settings.